Makefile

```diff
@@ -410,7 +410,6 @@ deploy/integreatly-rhmi-cr.yml:
 	sed "s/INSTALLATION_PREFIX/$(INSTALLATION_PREFIX)/g" | \
 	sed "s/INSTALLATION_SHORTHAND/$(INSTALLATION_SHORTHAND)/g" | \
 	sed "s/SELF_SIGNED_CERTS/$(SELF_SIGNED_CERTS)/g" | \
-	sed "s/IN_PROW/'$(IN_PROW)'/g" | \
```

Comment:
Hi @carlkyrillos as we are doing a separate ticket we should keep this line in for now as it's required by the existing logic.

Reply:
Hi @laurafitzgerald I added the line back to the Makefile and re-verified that the fix is still working as expected, thank you for catching that!